components/input-textUt/text-ut.html

```diff
@@ -0,0 +1,99 @@
+            <li><a href="/components/html/alert.html">Alert</a></li>
+            <li><a href="/components/html/avatar.html">Avatar</a></li>
+            <li><a href="/components/html/badge.html">Badge</a></li>
+            <li><a href="/components/html/button.html">Button</a></li>
+            <li><a href="/components/html/card.html">Card</a></li>
+            <li><a href="/components/html/image.html">Image</a></li>
+            <li><a href="/components/html/input.html">Input</a></li>
+            <li><a href="#">Text Utilities</a></li>
+            <li><a href="/components/html/list.html">Lists</a></li>
+            <li><a href="/components/html/nav.html">Navigation-Desktop</a></li>
+            <li><a href="/components/html/modal.html">Modal</a></li>
+            <li><a href="/components/html/rating.html">Rating</a></li>
+            <li><a href="/components/html/toast.html">Toast/Snackbar</a></li>
+            <li><a href="/components/html/card.html">Simplified Grid</a></li>
+        </ul>
+    </div>
+
+    <div class="Component-container">
+
+        <!---------Text-utilities componenet-------->
+        <div class="text pb2 " id="text-ut">
+            <h2 class="pb1 h2 comp-title">Text utilities</h2><br>
+            <p class="pb2">Text of variant styles and sizes can be used according to their respective class names.</p>
+            <h3>Example</h3>
+        </div>
+
+        <h2 class="grey mb2">Headings</h2>
+        <div class="typography-container">
+            <h1 class="h1">Heading 1</h1>
+            <h2 class="h2">Heading 2</h2>
+            <h3 class="h3">Heading 3</h3>
+            <h4 class="h4">Heading 4</h4>
+            <h5 class="h5">Heading 5</h5>
+        </div>
+
+        <div>
+            <iframe id="frame"
+                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=vscode&wt=none&l=auto&width=680&ds=true&dsyoff=61px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Ch1%2520class%253D%2522h1%2522%253EHeading%25201%253C%252Fh1%253E%250A%2520%253Ch2%2520class%253D%2522h2%2522%253EHeading%25202%253C%252Fh2%253E%250A%2520%253Ch3%2520class%253D%2522h3%2522%253EHeading%25203%253C%252Fh3%253E%250A%2520%253Ch4%2520class%253D%2522h4%2522%253EHeading%25204%253C%252Fh4%253E%250A%2520%253Ch5%2520class%253D%2522h5%2522%253EHeading%25205%253C%252Fh5%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
+                style=" height: 200px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+        </div>
+
+
+        <h2 class="grey mb2">Small text, gray text and center text</h2>
+
+        <div class="typography-container">
+            <p>This is a paragraph</p>
+            <p class="grey-text">This is a grey text</p>
+            <small>This is small text</small>
+            <center>This is centered text</center>
```

Comment:
this is not the right method to do using HTML tags to small or center text, rather you should give a CSS class. How the user is supposed to access it!

Reply:
I have made the changes,thankyou for review